### ISSUE TITLE
Sentinel-2 "eo:cloud_cover" doesn't match between v0 and v1

### ISSUE THREAD

**Issue by waltersdan** [Element84/earth-search#5]
When querying Sentinel-2 l2a imagery from these two collections:
https://earth-search.aws.element84.com/v0/collections/sentinel-s2-l2a-cogs
https://earth-search.aws.element84.com/v1/collections/sentinel-2-l2a
I can get the exact same images, except the property "eo:cloud_cover" is different for matching images. 

For example, these are the eo:cloud_cover values for a set of images from v0:
56.36  
87.25 
100.    
97.13
And the same images from v1:
78.148019 
92.783248 
99.999213 
86.64819
I have verified they are the exact same images and have the exact same pixel values for several bands including SCL. The only difference seems to be the metadata

**Comment by tnigon** [comment]
I've noticed this as well. Although I do not know for sure, I'm presuming it has to do with Earthsearch V1 using the most recent version of Sen2Cor, [which was released on December 6, 2022](https://step.esa.int/main/snap-supported-plugins/sen2cor/sen2cor-v2-11/).

I'd be interested in confirmation of this, as well as any other nuances on the topic.

**Comment by matthewhanson** [comment]
Yes, this may be an issue with the fact that the processing baseline between the L2 scene in Earth Search and the linked to L1 scene may have different processing baselines.  With the new reprocessing happening by ESA we hopefully can get this fixed.  Stay tuned to an update on the mailing list (and will post an issue in this repo).

**Comment by ircwaves** [comment]
@waltersdan -- Please check my thoughts here, and re-open if I'm off base:

1. Matt's call out that the baselines were changing at the time of your posting here sufficiently explains the issue encountered
2. I don't think there is any change needed for earthsearch, as this is an upstream processing change.
